sunpy/map/mapbase.py

```diff
@@ -975,15 +975,32 @@ def center(self):
     def shifted_value(self):
         """The total shift applied to the reference coordinate by past applications of
         `~sunpy.map.GenericMap.shift`."""
+        warn_deprecated(
+            '`sunpy.map.GenericMap.shifted_value` is deprecated and will be removed in sunpy 4.1. '
+            'Use `sunpy.map.GenericMap.meta.modified_items` to see how the '
+            'reference coordinate has been modified.'
+        )
         return self._shift
 
+    @deprecated('4.0', alternative='`sunpy.map.GenericMap.shift_reference_coord`')
     @u.quantity_input
     def shift(self, axis1: u.deg, axis2: u.deg):
+        GenericMap.shift.__doc__ = f"{GenericMap.shift_reference_coord.__doc__}"
```

Comment:
```suggestion
        GenericMap.shift.__doc__ = GenericMap.shift_reference_coord.__doc__
```
The format string isn't actually doing anything??